client/src/controllers/CountController.ts

```diff
@@ -0,0 +1,66 @@
+import { Controller } from '@hotwired/stimulus';
+import { ngettext } from '../utils/gettext';
+
+export class CountController extends Controller<HTMLFormElement> {
+  static targets = ['total', 'label'];
+  static values = {
+    containerBody: { type: String, default: 'body' },
+    min: { type: Number, default: 0 },
+    find: {
+      type: String,
+      default: '.error-message,.help-critical',
+    },
+    total: { type: Number, default: 0 },
+  };
+
+  declare containerValue: string;
+  declare minValue: number;
+  declare findValue: string;
+  declare totalValue: number;
+
+  declare readonly activeClass: string;
+  declare readonly hasActiveClass: string;
+  declare readonly hasLabelTarget: string;
+  declare readonly hasTotalTarget: string;
+  declare readonly labelTarget: HTMLElement;
+  declare readonly totalTarget: HTMLElement;
+
+  connect() {
+    // console.log("connected aman world");
+    setTimeout(() => {
+      console.log('connected CountController after load. ');
+      this.count();
+    }, 2000);
+  }
+
+  count() {
+    const containerElement = document.querySelector(
+      this.containerValue || 'body',
+    );
+    if (containerElement) {
+      this.totalValue = containerElement.querySelectorAll(
+        this.findValue,
+      ).length;
+      console.log(this.totalValue);
+      console.log(containerElement.querySelectorAll(this.findValue));
+    }
+  }
+
+  totalValueChanged(total) {
+    if (this.hasActiveClass) {
+      this.element.classList.toggle(this.activeClass, total > 0);
```

Comment:
```suggestion
      this.element.classList.toggle(this.activeClass, total > this.minValue);
```

If we are going to use the `minValue` we should consider it in all the logic in this method.

However, I recommend that we leave that feature to a future enhancement, it's really a nice to have and maybe just making this initial migration too complex.